src/app/component/dashboard/dashboard.component.ts

```diff
@@ -36,19 +30,10 @@ export class DashboardComponent implements OnInit,OnDestroy {
     public questionService: QuestionService,
   ){}
   
-  mapJSONData(){
-    // this.title = this.quizData?.QuizType?.title;
-    // this.image = this.quizData?.QuizType;
-    // this.description = this.quizData?.QuizType;
-    // console.log("title",this.title)
-    // console.log("image",this.image)
-    // console.log("description",this.description)
+  formatData() {
+    console.log(this.QuizType); 
   }
-  // next(){
-  //   this.imagesindex++;
-  //   this.imagessindex++;
-  //   this.imagesssindex++;
-  // }
+
   quizname(title:any){
     console.log(title);
     this.questionService.selectedQuizType = title;
```

Comment:
I guess service is not in use please remove this

Reply:
yes i removed it